src/main/java/com/prgms/voucher/voucherproject/domain/PercentDiscountVoucher.java

```diff
@@ -0,0 +1,31 @@
+package com.prgms.voucher.voucherproject.domain;
+
+import com.prgms.voucher.voucherproject.domain.Voucher;
+
+import java.util.UUID;
+
+public class PercentDiscountVoucher implements Voucher {
+    private final UUID voucherId;
+    private final long percent;
+
+
+    public PercentDiscountVoucher(UUID voucherId, long percent) {
+        this.voucherId = voucherId;
+        this.percent = percent;
+    }
+
+    @Override
+    public UUID getVoucherId() {
+        return null;
+    }
+
+    @Override
+    public long discount(long beforeDiscount) {
+        return beforeDiscount * (percent/100);
+    }
+
+    @Override
+    public String toString() {
+        return "| UUID:" + getVoucherId() + "  | VoucherType: PercentVoucher | percent:" + percent +" |";
```

Comment:
띄어쓰기가 빠져있어서 코드 제안 드렸어요.
```suggestion
        return "| UUID:" + getVoucherId() + "  | VoucherType: PercentVoucher | percent:" + percent + " |";
```